### ISSUE TITLE
Backup only works for 'local' or 's3' but not both

### ISSUE THREAD

**Issue by ryancwalsh** [spatie/laravel-backup#551]
In my `laravel-backup.php`, I've had:

```
'disks' => [
               'local',
                's3'
            ]
```

But after upgrading to Laravel 5.5 and lavavel-backup 5.0.4 and renaming `laravel-backup.php` to `backup.php`, now only my "local" backup is working.

I get no error messages, but only "local" works.

But if I comment out "local", then "s3" starts working.

Everything used to work perfectly until my upgrade.

I will continue to try to figure out the problem, but I figured I'd share with you in case this is a real problem or in case you know what I could have done wrong. Thanks.

**Comment by ryancwalsh** [comment]
Actually, it seems like I was incorrect about the problem. My local environment seems to successfully back up to local and S3 in the same run. 

But my production environment seems *not* to load `config/backup.php` even though it exists. So it never finds that I want to back up to S3.

To debug, I ran `vim vendor/spatie/laravel-backup/src/BackupServiceProvider.php` and then added `dd($this->app['config']);` after `$this->mergeConfigFrom(__DIR__.'/../config/backup.php', 'backup');` and then ran `php artisan backup:run --only-db`.

I did see "s3" in the config, but I noticed a "backup" array nested within a "laravel-backup" array:

```
 "laravel-backup" => array:4 [
      "backup" => array:4 [
```

That seems like a clue because when I do the same on my local environment, I see "backup" as an array within "backup".

But I'm still super confused.
